src/features/auth/api/instance.ts

```diff
@@ -0,0 +1,29 @@
+import axios, { Method } from 'axios';
+
+// todo add API_URL
+const BASE_URL = '/mock';
+
+const instance = axios.create({ baseURL: BASE_URL });
+
+export interface RequestConfig {
+  endpoint: string;
+  method: Method;
+  body?: { [key: string]: string };
+}
+
+export function createFormData(obj: { [key: string]: string } = {}) {
```

Comment:
해당 함수의 파라미터 인자인 'obj'의 타입은 Record로 선언해 주시는 게 더 좋아보입니다!

```suggestion
export function createFormData(obj: Record<string, string> = {}) {
```

어떻게 생각하시나요? 🙇‍♂️